integration test failures in develop

started Friday with https://jenkins.dataverse.org/job/IQSS-dataverse-develop/938/consoleFull

`[ERROR]   AccessIT.testRequestAccess:523 expected:<400> but was:<403>`

--- a/src/test/java/edu/harvard/iq/dataverse/api/AccessIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/AccessIT.java
@@ -520,7 +520,7 @@ public class AccessIT {
 
         listAccessRequestResponse = UtilIT.getAccessRequestList(tabFile3IdRestrictedNew.toString(), apiTokenRando);
         listAccessRequestResponse.prettyPrint();
-        assertEquals(400, listAccessRequestResponse.getStatusCode());
+        assertEquals(403, listAccessRequestResponse.getStatusCode());
 
         Response rejectFileAccessResponse = UtilIT.rejectFileAccessRequest(tabFile3IdRestrictedNew.toString(), "@" + apiIdentifierRando, apiToken);
         assertEquals(200, rejectFileAccessResponse.getStatusCode());

